stable/memcached/README.md

```diff
@@ -54,6 +54,7 @@ The following table lists the configurable parameters of the Memcached chart and
 | `extraVolumes`            | Volume definitions to add as string | Un-set                                              |
 | `kind`                    | Install as StatefulSet or Deployment | StatefulSet                                        |
 | `podAnnotations`          | Map of annotations to add to the pod(s) | `{}`                                            |
+| `customLabels`            | Custom Labels to be applied to statefulset | Un-set                                   |
```

Comment:
I think podLabels is a better name here. This was also the key used by another chart for the same thing. Could you rename?

Reply:
Sure thing